### ISSUE TITLE
Cannot open shared object file

### ISSUE THREAD

**Issue by Spone** [Gargron/blurhash#22]
Hello, I'm having the following error in my CI, right after running `bundle install`:

```
LoadError: Could not open library '/builds/my-project/vendor/bundle/ruby/3.1.0/gems/blurhash-0.1.6/lib/../ext/blurhash/encode.so': /builds/my-project/vendor/bundle/ruby/3.1.0/gems/blurhash-0.1.6/lib/../ext/blurhash/encode.so: cannot open shared object file: No such file or directory
```

I'm using Gitlab CI, with Ruby `3.1.3` and Blurhash `0.1.6`.

I'm not sure why this happens since I apparently didn't change anything on my side, so I'm posting here in case other people have the same issue. I'll reply if I find a fix.

**Comment by Spone** [comment]
And I can confirm that the `/builds/my-project/vendor/bundle/ruby/3.1.0/gems/blurhash-0.1.6/lib/../ext/blurhash/encode.so` file does not exist (I used the debug feature to log in to the CI runner and list the files in the `blurhash` directory).

I don't see any error message when the gem is installed:
```
Fetching blurhash 0.1.6
Installing blurhash 0.1.6 with native extensions
```

**Comment by renchap** [comment]
This is caused by recent Rubygems, see https://github.com/Gargron/blurhash/pull/19

It has been fixed in `master` (by #20), we need @Gargron to release a new version.

**Comment by Spone** [comment]
Thanks a lot @renchap, I'm changing my Gemfile to use master, while waiting for the new release.

```rb
# Gemfile
gem "blurhash", github: "Gargron/blurhash"
```
